code/game/gamemodes/game_mode.dm

```diff
@@ -246,12 +240,47 @@
 							//			recommended_enemies if the number of people with that role set to yes is less than recomended_enemies,
 							//			Less if there are not enough valid players in the game entirely to make recommended_enemies.
 
+// Just the above proc but for alive players
+/datum/game_mode/proc/get_alive_players_for_role(role, override_jobbans = FALSE)
+	var/list/players = list()
+	var/list/candidates = list()
+
+	var/roletext = get_roletext(role)
+
+	// Assemble a list of active players without jobbans.
+	for(var/mob/living/carbon/human/player in GLOB.player_list)
+		if(player.client && !(locate(player) in SSafk.afk_players))
+			if(!jobban_isbanned(player, ROLE_SYNDICATE) && !jobban_isbanned(player, roletext))
+//				if(player_old_enough_antag(player.client, role))
+				players += player
+
+	// Shuffle the players list so that it becomes ping-independent.
+	players = shuffle(players)
+
+	// Get a list of all the people who want to be the antagonist for this round, except those with incompatible species
+	for(var/mob/living/carbon/human/player in players)
+		if(!player.client.skip_antag)
+			if((role in player.client.prefs.be_special) && !(player.client.prefs.active_character.species in protected_species))
+				player_draft_log += "[player.key] had [roletext] enabled, so we are drafting them."
```

Comment:
who is we? and what are they being drafted into?

Reply:
An old comment, and they are being put into the "draft" of becoming a new antag